src/sys/event.rs

```diff
@@ -79,16 +78,18 @@ pub enum EventFilter {
 }
 
 #[cfg(any(target_os = "macos", target_os = "ios",
-          target_os = "freebsd", target_os = "dragonfly"))]
+          target_os = "freebsd", target_os = "dragonfly",
+          target_os = "openbsd"))]
 pub type type_of_event_flag = u16;
-#[cfg(any(target_os = "netbsd", target_os = "openbsd"))]
+#[cfg(any(target_os = "netbsd"))]
 pub type type_of_event_flag = u32;
 libc_bitflags!{
     pub flags EventFlag: type_of_event_flag {
         EV_ADD,
         EV_CLEAR,
         EV_DELETE,
         EV_DISABLE,
+        #[cfg(not(target_os = "openbsd"))]
```

Comment:
Sounds good. But please change this to a whitelist anyways. And add a `TODO` here that talks about adding it once OpenBSD is released (and name a specific version number so we don't need to look back in time to figure out this comment).